app/views/cases/offender_sar/_date_of_birth.html.slim

```diff
@@ -0,0 +1,3 @@
+tr.date-of-birth
+  th = t('common.case.date_of_birth')
+  td = l case_details.date_of_birth, format: :default
```

Comment:
As per comment for `_case_details.html.slim`, does this partial need to exist?

Reply:
It's the way everything is built